bower.json

```diff
@@ -19,17 +19,35 @@
   },
   "main": "paper-toast.html",
   "dependencies": {
-    "iron-a11y-announcer": "PolymerElements/iron-a11y-announcer#^1.0.0",
-    "iron-overlay-behavior": "PolymerElements/iron-overlay-behavior#^1.0.9",
-    "iron-fit-behavior": "PolymerElements/iron-fit-behavior#^1.1.0",
-    "polymer": "Polymer/polymer#^1.5.0"
+    "iron-a11y-announcer": "PolymerElements/iron-a11y-announcer#2.0-preview",
+    "iron-overlay-behavior": "PolymerElements/iron-overlay-behavior#2.0-preview",
+    "polymer": "Polymer/polymer#^2.0.0-rc.1"
```

Comment:
Not sure I understand. This PR has the same code running for both Polymer 1.0 and Polymer 2.0. There's no code changes here so either neither version uses any direct code from `iron-fit-behaviour`, and this dependency isn't need, or both are, and then it's needed in both